package.json

```diff
@@ -1,17 +1,18 @@
 {
-  "name": "@abi-software/gallery",
-  "version": "0.3.1",
+  "name": "@/abi-software/gallery",
```

Comment:
I don't think that this is the correct way of naming the package, I think the original name is correct.

Reply:
@Tehsurfer Can you please correct this please?

Reply:
Should be fixed now